README.md

```diff
@@ -69,7 +71,7 @@ If you have live server extension
 
 👥 Authors 
 
-- GitHub: https://github.com/theresetuyi
+- GitHub:  https://theresetuyi.github.io/porfolio/
```

Comment:
- [OPTIONAL] Could you put the link to your deployed portfolio under the ** _Live Demo_** Section of your _**README.md**_ file? You have put it as one of your social media links under the _**Author**_ section. Ideally, links under the _**Author**_ section should be your social media links. For example, the GitHub link should be the link to your GitHub homepage. 👍 

#### Hint:
- Change this GitHub link to what it was before
- Create a **_Live Demo_** section
- Put the deployed version of your portfolio under the **_Live Demo_** Section 👍